docker/inference/dockerfile.ctr

```diff
@@ -44,7 +46,7 @@ RUN ln -s /usr/bin/python3 /usr/bin/python
 # Install multiple packages
 RUN pip install cupy-cuda115 nvidia-pyindex pybind11 pytest protobuf transformers==4.12 tensorflow-metadata 
 RUN pip install betterproto cachetools graphviz nvtx scipy sklearn
-RUN pip install numba --no-deps
+RUN pip install pandas numba==0.55.1 numpy==1.21.5
```

Comment:
Do we need to fix versions here?

Reply:
For now we need to change them before installin Merlin Software. We need to update Merlin Software prerequisites